Fix build on macOS 11 and earlier whose readlink doesn't have a -f flag.
https://github.com/wxWidgets/wxWidgets/issues/25675
--- configure.orig	2025-05-25 12:15:56.000000000 -0500
+++ configure	2025-08-01 05:10:49.000000000 -0500
@@ -34271,19 +34271,20 @@
             DYLIB_RPATH_POSTLINK="${HOST_PREFIX}install_name_tool -id \$@ \$@"
             cat <<EOF >change-install-names
 #!/bin/sh
+libdir=\$(cd lib ; pwd -P)
 libnames=\$(cd lib ; ls -1 libwx*${WX_RELEASE}.dylib)
 changes=''
 for dep in \${libnames} ; do
-    target=\$(readlink -f \${4}/\${dep})
-    changes="\${changes} -change \${target} \${3}/\${dep}"
+    target=\$(readlink \${4}/\${dep}) || exit \$?
+    changes="\${changes} -change \${libdir}\${target} \${3}/\${dep}"
 done
 for i in \${libnames} ; do
-    lib=\$(readlink -f \${1}/\${i})
-    ${HOST_PREFIX}install_name_tool \${changes} -id \${3}/\${i} \${lib}
+    lib=\$(readlink \${1}/\${i}) || exit \$?
+    ${HOST_PREFIX}install_name_tool \${changes} -id \${3}/\${i} \${libdir}/\${lib} || exit \$?
 done
 
 if test -f "\${2}/wxrc-${WX_RELEASE}" ; then
-    ${HOST_PREFIX}install_name_tool \${changes} \${2}/wxrc-${WX_RELEASE}
+    ${HOST_PREFIX}install_name_tool \${changes} \${2}/wxrc-${WX_RELEASE} || exit \$?
 fi
 EOF
             chmod +x change-install-names
